activiti-spring-conformance-tests/activiti-spring-conformance-set0/src/test/java/org/activiti/spring/conformance/set0/ProcessInstanceOperationsTest.java

```diff
@@ -56,20 +57,22 @@ public void shouldBeAbleToStartAndDeleteProcessInstance() {
         assertThat(processInstance.getName()).isEqualTo("my-process-instance-name");
 
 
-        assertThat(collectedEvents).extracting(RuntimeEvent::getEventType).containsExactly(
+        assertThat(RuntimeTestConfiguration.collectedEvents).extracting(RuntimeEvent::getEventType).containsExactly(
                 ProcessRuntimeEvent.ProcessEvents.PROCESS_CREATED,
                 ProcessRuntimeEvent.ProcessEvents.PROCESS_STARTED,
                 BPMNActivityEvent.ActivityEvents.ACTIVITY_STARTED,
                 BPMNActivityEvent.ActivityEvents.ACTIVITY_COMPLETED,
                 BPMNSequenceFlowTakenEvent.SequenceFlowEvents.SEQUENCE_FLOW_TAKEN,
-                BPMNActivityEvent.ActivityEvents.ACTIVITY_STARTED);
+                BPMNActivityEvent.ActivityEvents.ACTIVITY_STARTED,
+                TaskRuntimeEvent.TaskEvents.TASK_CREATED);
 
-        collectedEvents.clear();
+        cleanUp();
```

Comment:
@CTI777, I know that currently `cleanUp` contains only `RuntimeTestConfiguration.collectedEvents.clear()`, but as it's annotated with `@Before` we may add other setup configurations there in the future. For that reason, I prefer to avoid calling directly `cleanUp` from inside a test method. You can encapsulate `RuntimeTestConfiguration.collectedEvents.clear()` inside another method `clearEvents` to get similar behavior. Does it make sense?